src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -189,6 +189,7 @@ def save_pretrained(
         save_directory: Union[str, os.PathLike],
         safe_serialization: bool = True,
         variant: Optional[str] = None,
+        max_shard_size: Union[int, str] = "10GB",
```

Comment:
can we leave the default at `None`, 5GB is the recommended default size we want to have across all huggingface libraries, we kept is at 10GB so that we won't start to automatically shard sdxl checkpoints. I don't think we should change this default value for our text encoders now

Reply:
Okay, let me open a quick follow-up PR